package.json

```diff
@@ -11,12 +11,14 @@
   "private": false,
   "scripts": {
     "build": "ember build --environment=production",
+    "lint": "run-p --aggregate-output -c -l lint:hbs lint:js",
     "lint:hbs": "ember-template-lint . --fix",
     "lint:js": "eslint . --fix",
     "lint:js:check": "eslint .",
     "start": "ember serve",
-    "test": "ember test",
-    "test:coverage": "COVERAGE=true run-s -l test",
+    "test": "run-p --aggregate-output -l lint test:ember",
+    "test:ember": "ember test",
+    "test:coverage": "COVERAGE=true ember test",
```

Comment:
The official installer add `npm-run-all` what we already had in this project... the tool added an aggregated `lint`, also changed test tasks, I had to a little bit align with our implementation.